test/Driver/Dependencies/Inputs/chained-additional-kinds/main.swift

```diff
@@ -1,3 +0,0 @@
-# Dependencies after compilation:
-depends-top-level: [a]
-provides-dynamic-lookup: [z]
```

Comment:
Must admit I'm chuckling at all the deletions! I hope that someday we do something to make sure we cover the same cases. EDIT: oh, sorry, I didn't see that you replaced these with the equivalent binaries! Thanks!!

Reply:
No, look at the commit history. One of the commits deletes a pile of input files which are actually not used at all.